mp/src/game/shared/momentum/mom_usermessages.cpp

```diff
@@ -54,5 +54,7 @@ void RegisterUserMessages(void)
     usermessages->Register("LobbyUpdateMsg", 9);
     usermessages->Register("SpecUpdateMsg", 17);
 
+	usermessages->Register("DamageIndicator", -1);
```

Comment:
Use spaces instead of tabs.

Reply:
You can also use 13 here instead of -1, because you're passing a byte (1) and 3 floats (3 * 4 bytes) every time.

Reply:
I put this as -1 because the size of the message seems to be variable - the size is 10 bytes usually, but in rare cases it can be 7-9 bytes when the rocket explodes at a coordinate close to 0:
```
User Msg 'DamageIndicator': 10 bytes written, expected 13
User Msg 'DamageIndicator': 9 bytes written, expected 13
```

Reply:
Sounds fine then, main concern is still spacing